examples/discord_links.py

```diff
@@ -3,46 +3,50 @@
 
 from coc.ext import discordlinks
 
-client = discordlinks.login(os.environ["LINKS_API_USERNAME"], os.environ["LINKS_API_PASSWORD"])
-
 
 async def main():
+    client = discordlinks.login(os.environ["LINKS_API_USERNAME"],
```

Comment:
Should there be an await somewhere here?

Reply:
You created a synchronous wrapper for the DiscordLinks class. We can remove the wrapper and make it async if you like. It looks like it's finding the current event loop using the default parameter for loop.